src/aml/ctm.py

```diff
@@ -80,12 +81,17 @@ def get_aspect_words(self, aspect_id, nwords): return self.mdl.get_word_distribu
 
     def infer_batch(self, reviews_test, h_ratio, doctype):
         reviews_test_ = []
-        reviews_aspects = []
+        reviews_aspects: List[List[List[str]]] = []
         for r in reviews_test:
             r_aspects = [[w for a, o, s in sent for w in a] for sent in r.get_aos()]  # [['service', 'food'], ['service'], ...]
-            if len(r_aspects[0]) == 0: continue  # ??
-            if random.random() < h_ratio: r_ = r.hide_aspects()
-            else: r_ = r
+
```

Comment:
@3ripleM seems no change. pls only push the required lines of changes. also one line pls